src/avro_util.erl

```diff
@@ -24,7 +24,8 @@
 -module(avro_util).
 
 %% API
--export([ canonicalize_aliases/2
+-export([ avro_schema_compatible/2
```

Comment:
I better name is perhaps `is_compatible/2`, and please expose this API through avro.erl